docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -478,10 +478,44 @@ Limitations:
 5.0 Model run requests
 ++++++++++++++++++++++
 
-.. todo::
+.. list-table:: Model run plan as of October 4, 2024
+  :header-rows: 1
 
-  Fill in this section as we continue to work
+  * - Number
+    - Run
+    - Status
+    - Priority
+    - Number of draws
+    - Population size per draw
+    - Note
+  * - 1
+    - Wave I Pakistan maternal V&V
```

Comment:
You can usually do all the locations as a single model "run". So just call is "Wave 1 Maternal (or pregnancy?) V&V" and then in the note include that we are doing XYZ locations. We could consider adding a column in the table for location if we anticipate changing locations for different runs often! In the past we rarely have done anything other than all locations.